src/c++/perf_analyzer/genai-perf/genai_perf/parser.py

```diff
@@ -400,21 +405,10 @@ def _add_endpoint_args(parser):
     endpoint_group.add_argument(
         "--endpoint-type",
         type=str,
-        choices=["chat", "completions"],
-        required=False,
-        help=f"The endpoint-type to send requests to on the "
-        'server. This is only used with the "openai" service-kind.',
-    )
-
-    endpoint_group.add_argument(
-        "--service-kind",
-        type=str,
-        choices=["triton", "openai"],
```

Comment:
I'm glad you got rid of this. It would have been very confusing for a user to know to use openai service kind for triton generate.

Reply:
yeah, I debated - but felt that it really would be confusing to continue with it.